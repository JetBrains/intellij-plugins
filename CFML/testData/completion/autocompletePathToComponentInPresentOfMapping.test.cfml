<cfset a = createObject("myfolde<caret>")>