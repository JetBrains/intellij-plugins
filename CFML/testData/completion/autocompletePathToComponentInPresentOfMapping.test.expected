<cfset a = createObject("myfolder.subfolder.")>